packages/react-accordion/src/components/Accordion/Accordion.types.ts

```diff
@@ -11,8 +11,12 @@ export type AccordionToggleEventHandler = (event: AccordionToggleEvent, data: Ac
 type AccordionCommons = {
   /**
    * Indicates if keyboard navigation is available
+   * By default circular navigation is available,
+   * if 'linear' is provided than there's no circular navigation
+   *
+   * @default false
    */
-  navigable: boolean;
+  navigable: boolean | 'linear';
```

Comment:
@layershifter  any better API ideas in your experience ?

Reply:
Looking at this boolean union with string enum, I would say **let's just break the API and call this prop `navigation: linear | circular` **